docker-compose.yml

```diff
@@ -10,8 +10,11 @@ services:
         - no_proxy
     image: hmctspublic.azurecr.io/payment/api
     environment:
-      - auth.provider.service.client.baseUrl=http://service-auth-provider-api:8080/
-      - auth.idam.client.baseUrl=https://idam-test.dev.ccidam.reform.hmcts.net
+      - auth.provider.service.client.baseUrl=http://service-auth-provider-api:8489
```

Comment:
please check with 8080 port instead of 8489

Reply:
Checked with 8080, the payment app had connection refused. It is looking for service running at 8489